docs/content/common-subs/kubestellar-test-apache.md

```diff
@@ -0,0 +1,62 @@
+<!--kubestellar-test-apache-start-->
+Now, let's check that the deployment was created in the kind **ks-edge-cluster1** cluster (it may take up to 30 seconds to appear):
+```shell
+KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster1 get deployments -A
+```
+
+you should see output including:
+``` { .sh .no-copy }
+NAMESPACE           NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+my-namespace        my-first-kubestellar-deployment    1/1        1            1       6m48s
+```
+
+And, check the **ks-edge-cluster2** kind cluster for the same:
+```shell
+KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster2 get deployments -A
+```
+
+you should see output including:
+``` { .sh .no-copy }
+NAMESPACE           NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+my-namespace        my-first-kubestellar-deployment    1/1        1            1       7m54s
+```
+
+Finally, let's check that the workload is working in both clusters:
+For **ks-edge-cluster1**:
+```shell
+while [[ $(KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster1 get pod \
+  -l "app=common" -n my-namespace -o jsonpath='{.items[0].status.phase}') != "Running" ]]; do 
+    sleep 5; 
+  done;
+curl http://localhost:8094
+```
```

Comment:
This is the point at which my attempt to execute the Quickstart fails. I've gone through twice to make sure I didn't skip any steps (still possible), but executing `KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster1 get pod \
  -l "app=common" -n my-namespace -o jsonpath='{.items[0].status.phase}'` gives the error

```
error: error executing jsonpath "{.items[0].status.phase}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].status.phase}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":""}}
```

ks-edge-cluster1 log:
```
I1023 15:24:24.338879       1 reflector.go:307] Listing and watching *v2alpha1.EdgeSyncConfig from k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215
W1023 15:24:24.399611       1 reflector.go:376] k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215: failed to list *v2alpha1.EdgeSyncConfig: Get "https://kubestellar.core:4000/apis/edge.kubestellar.io/v2alpha1/edgesyncconfigs?limit=500&resourceVersion=0": dial tcp: lookup kubestellar.core on 10.96.0.10:53: no such host
E1023 15:24:24.399674       1 reflector.go:138] k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215: Failed to watch *v2alpha1.EdgeSyncConfig: failed to list *v2alpha1.EdgeSyncConfig: Get "https://kubestellar.core:4000/apis/edge.kubestellar.io/v2alpha1/edgesyncconfigs?limit=500&resourceVersion=0": dial tcp: lookup kubestellar.core on 10.96.0.10:53: no such host
```

ks-edge-cluster2 log:

```
I1023 15:25:53.899697       1 reflector.go:307] Listing and watching *v2alpha1.EdgeSyncConfig from k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215
W1023 15:25:53.956613       1 reflector.go:376] k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215: failed to list *v2alpha1.EdgeSyncConfig: Get "https://kubestellar.core:4000/apis/edge.kubestellar.io/v2alpha1/edgesyncconfigs?limit=500&resourceVersion=0": dial tcp: lookup kubestellar.core on 10.96.0.10:53: no such host
E1023 15:25:53.956738       1 reflector.go:138] k8s.io/client-go@v0.0.0-20230210192259-aaa28aa88b2d/tools/cache/reflector.go:215: Failed to watch *v2alpha1.EdgeSyncConfig: failed to list *v2alpha1.EdgeSyncConfig: Get "https://kubestellar.core:4000/apis/edge.kubestellar.io/v2alpha1/edgesyncconfigs?limit=500&resourceVersion=0": dial tcp: lookup kubestellar.core on 10.96.0.10:53: no such host
```

My `/etc/hosts`:

```
127.0.0.1	localhost
127.0.1.1	debian
192.168.122.144 kubestellar.core

# The following lines are desirable for IPv6 capable hosts
::1     localhost ip6-localhost ip6-loopback
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters

```